pkgs/applications/video/kaiscr/default.nix

```diff
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitea
+, gobject-introspection
+, gtk3
+}:
+
+buildPythonApplication rec {
+  pname = "KaiScr";
+  version = "unstable-2020-10-21";
+
+  src = fetchFromGitea {
+    domain = "notabug.org";
+    owner = "farooqkz";
+    repo = pname;
+    rev = "3ebb16d29075d09fe0bb69f5a00d57e107cae6c2";
+    sha256 = "sha256-JnDjExGCfDKCvgH+h5QwYaTzaQ+2YEAkiSFrlw9PGOc=";
+  };
+
+  # src is plain Python3 scripts.
+  format = "other";
+  dontUseSetuptoolsBuild = true;
+  dontUsePipInstall = true;
+  dontUseSetuptoolsCheck = true;
+
+  postPatch = ''
+    # kailive isn't marked as exec
+    chmod +x kailive.py
+  '';
+
+  buildInputs = [
+    gobject-introspection
+    gtk3
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp kailive.py $out/bin/kailive
+    cp kaiscr.py $out/bin/kaiscr
+  '';
+
+  checkPhase = ''
+    runHook preCheck
+    $out/bin/kaiscr -h
+    runHook postCheck
+  '';
```

Comment:
Weird thing to do inside a check phase. Can we use `testers.testVersion` instead?

Reply:
There's no `--version` and I think it'd be misleading to `testVersion` on the `--help` when it'd be doing the same thing as this. I'll make it `--help` though.